aspnet/2-structured-data/Models/DatastoreBookStore.cs

```diff
@@ -48,34 +56,11 @@ public static long ToId(this Key key)
             return (long)key.Path.First().Id;
         }
 
-        /// <summary>
-        /// Get the property from the dict and return null if it isn't there.
-        /// </summary>
-        /// <param name="properties"></param>
-        /// <param name="key"></param>
-        /// <returns></returns>
-        public static Property GetValue(this IDictionary<string, Property> properties, string key)
-        {
-            Property value;
-            bool found = properties.TryGetValue(key, out value);
-            return found ? value : null;
-        }
-
-        /// <summary>
-        /// Creates a new property iff value is not null.
-        /// </summary>
-        public static Property NewProperty(string value)
-        {
-            return null == value ? null : new Property() { StringValue = value };
-        }
-
-        public static Property NewProperty(DateTime? value)
-        {
-            return null == value ? null : new Property() { DateTimeValue = value };
-        }
+        // P2 What I really want is an ORM-like solution so that I don't

```

Comment:
Explicitly out of scope for now, though yes, an ORM solution would be nice. If we do one, I expect it would be in a separate library.